.github/workflows/build.yml

```diff
@@ -141,7 +141,7 @@ jobs:
       uses: docker/build-push-action@v5
       with:
         context: .
-        platforms: linux/amd64
+        platforms: linux/amd64,linux/arm64
```

Comment:
The base Ubuntu image should probably also have this.

Reply:
@christophebedard also added arm64 into base https://github.com/ros-tooling/setup-ros-docker/pull/70/commits/d463c20e6065cfba7eb9500b83649692670a739f.